Author: Fabian Greffrath <fabian+debian@greffrath.com>
Description: Fixes "format not a string literal and no format arguments
 [-Werror=format-security]" errors
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=643377

--- flac.orig/src/test_grabbag/cuesheet/main.c
+++ flac/src/test_grabbag/cuesheet/main.c
@@ -115,12 +115,12 @@ int main(int argc, char *argv[])
 	const char *usage = "usage: test_cuesheet cuesheet_file lead_out_offset [ cdda ]\n";
 
 	if(argc > 1 && 0 == strcmp(argv[1], "-h")) {
-		printf(usage);
+		printf("%s", usage);
 		return 0;
 	}
 
 	if(argc < 3 || argc > 4) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		return 255;
 	}
 
@@ -129,7 +129,7 @@ int main(int argc, char *argv[])
 		if(0 == strcmp(argv[3], "cdda"))
 			is_cdda = true;
 		else {
-			fprintf(stderr, usage);
+			fprintf(stderr, "%s", usage);
 			return 255;
 		}
 	}
--- flac.orig/src/test_grabbag/picture/main.c
+++ flac/src/test_grabbag/picture/main.c
@@ -211,12 +211,12 @@ int main(int argc, char *argv[])
 	const char *usage = "usage: test_pictures path_prefix\n";
 
 	if(argc > 1 && 0 == strcmp(argv[1], "-h")) {
-		printf(usage);
+		printf("%s", usage);
 		return 0;
 	}
 
 	if(argc != 2) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		return 255;
 	}
 
--- flac.orig/src/test_seeking/main.c
+++ flac/src/test_seeking/main.c
@@ -429,7 +429,7 @@ int main(int argc, char *argv[])
 	static const char * const usage = "usage: test_seeking file.flac [#seeks] [#samples-in-file.flac] [file.raw]\n";
 
 	if (argc < 2 || argc > 5) {
-		fprintf(stderr, usage);
+		fprintf(stderr, "%s", usage);
 		return 1;
 	}
 
